apps/openmw/mwclass/container.cpp

```diff
@@ -145,7 +145,7 @@ namespace MWClass
         // make key id lowercase
         std::string keyId = ptr.getCellRef().getKey();
         Misc::StringUtils::lowerCaseInPlace(keyId);
-        for (MWWorld::ContainerStoreIterator it = invStore.begin(); it != invStore.end(); ++it)
```

Comment:
@scrawl  Testing it here, steping through GDB, all fine.
The only thing that bothers me, is when you try to convert ConstIterator to Iterator you get a linking error instead of an compiling error. But that keeps the code smaller.